src/vm/eventpipefile.cpp

```diff
@@ -77,7 +77,8 @@ DWORD GetFileMinVersion(EventPipeSerializationFormat format)
 }
 
 EventPipeFile::EventPipeFile(StreamWriter *pStreamWriter, EventPipeSerializationFormat format) :
-    FastSerializableObject(GetFileVersion(format), GetFileMinVersion(format), format >= EventPipeSerializationFormat::NetTraceV4)
+    FastSerializableObject(GetFileVersion(format), GetFileMinVersion(format), format >= EventPipeSerializationFormat::NetTraceV4), 
+    m_pStreamWriter(pStreamWriter)
```

Comment:
nit: `m_pSerializer(nullptr)` as the member is not initialized in the ctor anymore.